docs/personal-data.md

```diff
@@ -34,46 +34,64 @@ Below a list of all the events in Stepup, together with their data types. The da
+| *contact_information*         |                          | contact information entered by an RA(A)                                                                                                              |
+| email_verification_nonce      |                          | the nonce that was sent to the user to verify the email address during registration                                                                  |
+| email_verification_required   |                          | whether email verification is required (deprecated)                                                                                                  |
+| email_verification_window     |                          | start and end times of the validity of the email_verification_nonce                                                                                  |
+| *identity_id*                 |                          | GUID. This is the unique identifier of an identity in Stepup. It is used internally in Stepup only.                                                  |
+| *identity_institution*        |                          | the identifier of the institution that the user belongs to. This is the value of the schacHomeOrganization attribute as received from the remote IdP |
+| institution                   |                          | the identifier of the institution to which the event applies                                                                                         |
+| *location*                    |                          | contact location entered by an RA(A)                                                                                                                 |
+| *name_id*                     |                          | The nameID in the Subject of the Assertion as received from the remote IdP                                                                           |
+| preferred_locale              |                          | The locate of the user                                                                                                                               |
+| *registration_authority_role* |                          | role identifier (1=RA or 2=RAA)                                                                                                                      |
+| registration_code             |                          | the registration code                                                                                                                                |
+| registration_requested_at     |                          | timestamp at which registration was requested                                                                                                        |
+| *second_factor_id*            | &#9745;                  | GUID. Unique identifier of the second factor                                                                                                         |
```

Comment:
The second_factor_id is not in the sensitive data stream right? I think you are mistaken with the second_factor_identifier (which can be a phone number, yubikey public key, gssp data, ..)